fibonacci_sequence.py

```diff
@@ -0,0 +1,7 @@
+x_1 ,x_2 = 0, 1
+
+while x_2 <= 1597:

```

Comment:
since this number is a constant we should not add that directly in the validation.
Please create a constant with that value first, then use that constant in the while